### ISSUE TITLE
How to add an id to the nodes?

### ISSUE THREAD

**Issue by felipe-brancaleone** [bmelnychuk/AndroidTreeView#81]
It's possible to do this?(add an id to the nodes) Because i'm getting those from the server, so the user will check some of then and send it to the server again. And in the server i need to know what are the items clicked by the user!

Thanks
Felipe

**Comment by bmelnychuk** [comment]
Not really, id is used for internal purposes, but instead you can add to model you represent as treeItem. You can always access it when working with node (expand/collapse/select)

**Comment by GeorgePractics23** [comment]
@felipe-brancaleone You have to create a subclass of TreeNode and here you can add the parameters that you need. I have created one, and I have added two variables, the nodeId and the parentId! Hope it will help you.
